test/e2e/testsuites/dynamically_provisioned_cmd_volume_tester.go

```diff
@@ -46,5 +48,10 @@ func (t *DynamicallyProvisionedCmdVolumeTest) Run(client clientset.Interface, na
 		defer tpod.Cleanup()
 		ginkgo.By("checking that the pod's command exits with no error")
 		tpod.WaitForSuccess()
+
+		err := ValidateAzVolumeAttachment(namespace, tpod, client, azDiskClient, t.StorageClassParameters, isV2Driver)
```

Comment:
Rather than skipping `AzVolume[Attachment]` validation inside this function, consider using an `if` statement here to decide whether or not to do the validation.